primitives/consensus/common/src/import_queue/buffered_link.rs

```diff
@@ -137,12 +138,17 @@ impl<B: BlockT> BufferedLinkReceiver<B> {
 	/// it is as if this method always returned `Poll::Pending`.
 	pub fn poll_actions(&mut self, cx: &mut Context, link: &mut dyn Link<B>) {
```

Comment:
This function would panic occasionally on exit because `Stream::poll_next` is called for a finished stream.